CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Braintree iOS Drop-in SDK - Release Notes
 
+## unreleased
+
+* Fix several UIKit deprecation warnings
```

Comment:
Why not specify the ones we changed?

Reply:
It might be a bit too much detail for the changelog, but here's what it would look like:

```
* Fix UIKit deprecation warnings for UIApplication's `statusBarOrientation` and `statusBarFrame` properties and UIViewController's `topLayoutGuide` and `bottomLayoutGuide` properties
```

What do you think? Too much info for the changelog?

Reply:
I'm for it. That's just me!